index.js

```diff
@@ -20,7 +24,8 @@ module.exports = function({ types: t }) {
           path,
           path.node.id.name,
           sourceFileNameFromState(state),
-          ...attributeNamesFromState(state)
```

Comment:
what's the significance of removing the `...`?

Reply:
tbh not much, just that the parameters for these functions got too long. Also adding the new `ignoreComponentsFromOption` after a spread operator seemed like it could be confusing. I don't feel really strongly about using `...` here, maybe the `ignoreComponentsFromOption` can come before the `sourceFileNameFromState` and the `...attributeNamesFromState(state)` can remain in the last position